sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/TelemetryItem.cs

```diff
@@ -60,10 +60,7 @@ internal partial class TelemetryItem
 
             SetAuthenticatedUserId(ref activityTagsProcessor);
             SetResourceSdkVersionAndIkey(resource, instrumentationKey);
-            if (AzMonList.GetTagValue(ref activityTagsProcessor.MappedTags, "sampleRate") is float sampleRate)
-            {
-                SampleRate = sampleRate;
-            }
+            SampleRate = sampleRate;
```

Comment:
set this property if sampleRate != 100.

Reply:
There is another overload in this file, please check the same information there.

Reply:
made the change.

Also checked all other references. This is the only place where we set the property.